build/commands/lib/test.js

```diff
@@ -123,6 +123,7 @@ const test = (passthroughArgs, suite, buildConfig = config.defaultBuildConfig, o
 
   if (config.targetOS === 'ios') {
     util.run(path.join(config.outputDir, "iossim"), [
+      "-d", "\"iPhone 14 Pro\"",
```

Comment:
shouldn't this be the oldest device we currently support?

Reply:
It doesn't really make a difference, simulators don't actually replicate the CPUs or device capabilities aside from feature specific stuff (Dynamic Island, etc.) which we won't test for in brave-core anyways